tools/collaboration.md

```diff
@@ -9,7 +9,7 @@ parent: Tools
 
 This document outlines the communications tools available for {{ site.foundation_name }} hosted projects and working groups.
 
-{{ site.foundation_name }} leverages the tools provided by the LFX Project Control Center (PCC) for maintaining committee member lists, scheduling meetings, configuring mailing lists, and making community meetings publicly available. The LFX Project Control Center is a tool built and maintained by the Linux Foundation for use by the {{ site.foundation_name }} and its hosted projects and working groups. Access to PCC for hosted projects and working groups for management of meetings and committees can be granted by making a support request [here][pcc access request instructions].
+{{ site.foundation_name }} leverages the tools provided by the LFX Project Control Center (PCC) for maintaining committee member lists, scheduling meetings, configuring mailing lists, and making community meetings publicly available. The LFX Project Control Center is a tool built and maintained by the Linux Foundation for use by the {{ site.foundation_name }} and its hosted projects and working groups. Access to PCC for hosted projects and working groups for management of meetings and committees can be granted by making a support request [here][pcc access request instructions]. It is recommended that, at a minimum, the TSC Chairperson have access to PCC; it may also be beneficial to have a second person with access in case the TSC Chairperson is unavailable and the project needs to reschedule a meeting. 
```

Comment:
this means that only the TSC and the back-up can:
Change committees
Change domainname settings
See cloudproviders (if used)
Create polls and surveys

Their is no public landing page. It is recommened to share the calendar URL somewhere else (e.g. on the website) and share the persons who can schedule meetings and can make LFX changes.

Reply:
@jmertic can you shine a light on the different permissions in LFX?

![image](https://github.com/user-attachments/assets/9e318629-a81d-42a4-9d37-0cd22506c70b)

Reply:
More information on the capabilites of LFX can be found - URL to the LF LFX docs -

Reply:
Hey @Sander3003 - I've included much of what you mentioned. I don't want to overload people with too much information, so I didn't add anything on the permissions and other capabilities. Let me know if you think that is ok.

Reply:
I added a link to the PCC docs overall, which I think maybe addresses the question? There is some fluidity on the permissions structure and tools overall, so I'm hesitant to redocument them here if that makes sense. LMK if this approach seems reasonable.

Reply:
I suggest to add the permissions topic to the LFx documentation. You can send a pull reques to their repo if you have the righst to do so (I can't): https://docs.linuxfoundation.org/lfx/project-control-center.

In that case, it his sufficient.